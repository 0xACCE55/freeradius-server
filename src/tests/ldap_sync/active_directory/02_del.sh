#!/bin/sh

if [ "$ACTIVE_DIRECTORY_TEST_SERVER " = " " ]; then
  echo ACTIVE_DIRECTORY_TEST_SERVER not defined
  exit 1;
fi

# If using a "remote" server, ssh key auth must be set up so
# samba-tool can be run as root to perform modifications

CMDSTART="sudo "
if [ "$ACTIVE_DIRECTORY_TEST_SERVER" != "127.0.0.1" ]; then
CMDSTART="ssh root@$ACTIVE_DIRECTORY_TEST_SERVER "
fi

${CMDSTART}samba-tool group delete dummy > /dev/null
${CMDSTART}samba-tool computer delete test_workstation > /dev/null
${CMDSTART}samba-tool user disable fred > /dev/null
${CMDSTART}samba-tool user delete fred > /dev/null
