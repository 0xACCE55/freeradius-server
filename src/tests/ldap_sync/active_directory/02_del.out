Delete CN=Deleted Objects,dc=example,dc=com CN=fred\0ADEL:oldid,CN=Deleted Objects,DC=example,DC=com fred 514 TRUE
